modules/react/common/spec/createCanvasTheme.spec.tsx

```diff
@@ -3,7 +3,7 @@ import {defaultCanvasTheme, createCanvasTheme, PartialCanvasTheme} from '../lib/
 import {shiftColor} from '../lib/theming/createCanvasTheme';
 import {deepMerge} from '../lib/utils/deepMerge';
 
-describe('createCanvasTheme', () => {
+describe.only('createCanvasTheme', () => {
```

Comment:
Is this `only` supposed to be here still?

```suggestion
describe('createCanvasTheme', () => {
```